zds/utils/templatetags/topics_sort.py

```diff
@@ -0,0 +1,13 @@
+from django import template
+from django.core.cache import cache
+
+
+register = template.Library()
+
+
+@register.filter("topics_sort")
+def topics_sort(topics):
+    """
+    :return: the topics sorted by last update (last updated first)
+    """
+    return sorted(topics, key=lambda topic: topic.update, reverse=True)
```

Comment:
Si jamais tu acceptes ma suggestion `Topic.last_update` il faut modifier cette ligne aussi. :)
```suggestion
    return sorted(topics, key=lambda topic: topic.last_update, reverse=True)
```